sosww/components/dynamo_db.py

```diff
@@ -316,6 +317,72 @@ def _build_scan_iterator(self, attrs=None, table_name=None, strict=True):
         return response_iterator
 
 
+    def batch_get_items_one_table(self, keys_list, table_name=None, max_retries=0, retry_wait_base_time=0.2):
+        """
+        Gets a batch of items from a single dynamo table.
+        Only accepts keys, can't query by other columns.
+
+        :param list keys_list: A list of the keys of the items we want to get.
+                               Ex: [{'hash_col': '1, 'range_col': 2}, {'hash_col': 3, 'range_col': 4}, ...]
+
+        Optional
+
+        :param str table_name:
+        :param int max_retries: If failed to get some items, retry this much times. Waiting between retries is
+                                multiplied by 2 after each retry, so `retries` shouldn't be a big number.
+                                Default is 1.
+        :param int retry_wait_base_time: Wait this much time after first retry. Will wait twice longer in each retry.
+        :return:
+        """
+
+        table_name = self._get_validate_table_name(table_name)
+
+        # Convert given keys to dynamo syntax
+        query_keys = [self.dict_to_dynamo(item) for item in keys_list]
+
+        batch_get_item_query = {
+            'RequestItems': {
+                table_name: {
+                    'Keys': query_keys
+                }
+                # Just so you know - you could add another query for another table here.
```

Comment:
give commented example of how to do that please.

Reply:
I'll just remove this comment...